services/utils/conditional-headers/src/main/java/org/eclipse/ditto/services/utils/headers/conditional/ConditionalHeadersValidator.java

```diff
@@ -95,13 +107,36 @@ public void checkConditionalHeaders(final Command command,
 
         checkIfMatch(command, currentETagValue);
         checkIfNoneMatch(command, currentETagValue);
+
     }
 
     private boolean skipPreconditionHeaderCheck(final Command command, @Nullable final EntityTag
             currentETagValue) {
         return currentETagValue == null &&
                 (Command.Category.DELETE.equals(command.getCategory()) ||
-                        Command.Category.QUERY.equals(command.getCategory()));
+                        Command.Category.QUERY.equals(command.getCategory())) || skipDueToRelation(command);
```

Comment:
I would appreciate making this statement clearer as now I have to know which binary operator has more weight here.
This would make it more clear:
```suggestion
        return (currentETagValue == null && (
                Command.Category.DELETE.equals(command.getCategory()) ||
                Command.Category.QUERY.equals(command.getCategory()))
        ) || skipDueToRelation(command);
```